build.sh

```diff
@@ -6,12 +28,27 @@ export DO_UPDATE
 export VERBOSE
 export QUIET
 
+HTML_CACHE=${HTML_CACHE:-.cache}
+export HTML_CACHE
+
+HTML_TEMP=${HTML_TEMP:-.temp}
+export HTML_TEMP
+
+HTML_OUTPUT=${HTML_OUTPUT:-output}
+export HTML_OUTPUT
+
 for arg in "$@"
 do
   case $arg in
+    -c|--clean)
+      rm -rf $HTML_CACHE
+      rm -rf $HTML_TEMP
+      exit 0
+      ;;
     -h|--help)
-      echo "Usage: $0 [-h|--help] [-n|--no-update] [-q|--quiet] [-v|--verbose]"
+      echo "Usage: $0 [-c|clean] [-h|--help] [-n|--no-update] [-q|--quiet] [-v|--verbose]"
```

Comment:
> Typo: clean => --clean

Thanks for catching that.

> Maybe separate usage lines for the different modes? Not sure the exact convention, but maybe this would be clearer…

OK, your suggestion looks to me that the correct conventional way to do it, so made it so